City-of-Helsinki/tilavarauspalvelu-ui
# Application flow fixes

## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Addresses the issues listed on the application flow ticket (TILA-3560), apart from ones to do with validating that the specific timespans correspond to the given amount and duration in the application
- Fixes page2 of the application flow in respect to the application sections, and selected reservation units

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of what's the fastest way to test your changes."

- Check out the application flow, comparing it to [the design in figma](https://www.figma.com/design/s987FF5VRRUNhDBvZJFPs9/HEL-Varaamo?node-id=4318-369406&t=fHO7a0Yu1DvlTBH6-1).
- Make an application with more than one sections and note, that you can now enter the primary/secondary timeslots and the form should all-in-all work as expected when filling in details for more than one section in one application. Using sections with different selections of spaces makes this easier to see at a glance.

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)"

- TILA-3560
